src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println(getName() + " started its work..");
```

Comment:
Same for other classes
```suggestion
        System.out.println("Bulldozer started its work..");
```

Reply:
Have to remove getName method and change to Bulldozer or anther name. Got you